src/Extra/ApiMediaFile.php

```diff
@@ -16,7 +16,11 @@
 use Symfony\Component\HttpFoundation\File\File;
 
 /**
+ * NEXT_MAJOR: Remove this file.
+ *
  * @final since sonata-project/media-bundle 3.21.0
```

Comment:
IMHO you can remove the `@final` information

Reply:
Does it matter? I will remove it once I do the manual merge

Reply:
Yes, because this information is for the user and can be very confusing. It could either be final or deprecated.

Reply:
I don't see how final since 3.21 and deprecated since 3.34 could confuse our users, but changed!.